send/openstack_projects

```diff
@@ -40,4 +40,3 @@ else
 fi
 
 exit $ERR_CODE
- 
```

Comment:
This is a not needed change. It can be replaced by the original script with the right end of the file.